requirements.txt

```diff
@@ -0,0 +1,11 @@
+inflect
```

Comment:
Once a user install these requirements, will they be able to run the readme quick start example
```
# Ner4Opt Model options: lexical, lexical_plus, semantic, hybrid
ner4opt = Ner4Opt(model='hybrid')
```
with any model option?

Reply:
Defaults to hybrid, I updated the readme accordingly.